src/ConfigManager.h

```diff
@@ -69,7 +69,7 @@ class ConfigManager {
     static G4double      GetEventWindow()                   { return Instance()->eventWindow_; }
     static G4bool        GetSNTimingOn()                    { return Instance()->snTimingOn_; }
     static G4String      GetTh2Name()                       { return Instance()->th2Name_; }
-    static G4bool        GetUseHDDetectorConfiguration()    { return Instance()->useHDDetectorConfiguration_; }
+    static G4String      GetDetectorConfiguration()         { return Instance()->detectorConfiguration_; }
```

Comment:
Always return non built-in data types by const reference, unless you have a *really* good reason to have to return by value.  Use `const G4String&`.

Reply:
Will revise this line. Thank you Mike!